python/requirements.txt

```diff
@@ -5,4 +5,5 @@ pydantic>=2.1
 pybars3
 google-generativeai
 openai
-python-dotenv
\ No newline at end of file
+python-dotenv
+huggingface_hub
```

Comment:
nit: don't think this needs to be part of this diff. But ok since I think you're adding it soon.